source/request-response/request_response_client.c

```diff
@@ -1259,6 +1731,78 @@ void s_aws_mqtt_request_operation_storage_init_from_options(
     storage->options.response_paths = storage->operation_response_paths.data;
 }
 
+static void s_log_request_response_operation(
+    struct aws_mqtt_rr_client_operation *operation,
+    struct aws_mqtt_request_response_client *client) {
+    struct aws_logger *log_handle = aws_logger_get_conditional(AWS_LS_MQTT_REQUEST_RESPONSE, AWS_LL_DEBUG);
+    if (log_handle == NULL) {
+        return;
+    }
+
+    struct aws_mqtt_request_operation_options *options = &operation->storage.request_storage.options;
+
+    AWS_LOGUF(
```

Comment:
Trivial: I've seen grouped logs like this elsewhere and just wondering if it would make sense to combine all these individual logs into a single log instead of a stream of a half dozen that may be broken up by other logs. No biggie since if we combined them, they'd probably take up the same amount of vertical space in the logs anyway.